client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,25 +28,26 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
-
-		// TODO: add from product.
-		const imageUrl = 'https://cldup.com/nKM0_KspYE.png';
+		const { productImage } = this.props;
+		const { title, description, price, currency, featuredImageId: productImageId } = product;
 
 		// TODO: make proper icon and store on some proper place.
 		const paypalButtonImageUrl = 'https://cldup.com/DoIAwrACBs.png';
 
 		return (
 			<div className="wpview-content wpview-type-simple-payments">
+				{ productImageId && <QueryMedia siteId={ siteId } mediaId={ productImageId } /> }
 				<div className="wpview-type-simple-payments__wrapper">
+				{ productImage &&
 					<div className="wpview-type-simple-payments__image-part">
 						<figure className="wpview-type-simple-payments__image-figure">
 							<img
 								className="wpview-type-simple-payments__image"
-								src={ imageUrl }
+								src={ productImage.URL }
```

Comment:
This URL points to the original image that can be potentially huge and yet it's always resized into a 200x200px rectangle. Using something like `MediaUtils.url( productImage, { size: 'medium' } )` would download a smaller version.